test/discriminating_facets_test.py

```diff
@@ -0,0 +1,89 @@
+import unittest
+from mlapi.model.facet import Facet
+from mlapi.question_generator import DiscriminatingFacetsAlgo
+import statistics
+
+class TestDiscriminatingFacets(unittest.TestCase):
```

Comment:
if we remove facet_c2, the facet stay discriminating.
 In this test, the target is to have 2 discriminating facets to test when I don't adjust parameters. 
Because in the 2nd test, I have only one discriminating facet. so the code execute the algo for the first time, and then adjust parameters and reexecute it one more time.